src/js/angular/clustermanagement/controllers.js

```diff
@@ -30,7 +30,7 @@ export const NodeState = {
     READ_ONLY: 'READ_ONLY',
     RESTRICTED: 'RESTRICTED',
     NO_CLUSTER: 'NO_CLUSTER',
-    DELETED: 'DELETED'
+    DELETED: 'Cluster was deleted on this node.'
```

Comment:
This is not a label. It is an object that is used as an Enum. Why did you change it's value?

Is this the correct NodeState value comming from the server!?